src/Redemption.js

```diff
@@ -235,6 +235,53 @@ export default class Redemption {
     return this.autoSubmittingState
   }
 
+  /**
+   * Increases redemption transaction fee.
```

Comment:
Worth noting here that the increase is done by a an increment equal to the initial redemption fee, but not a blocker.